plugins/lookup/sops.py

```diff
@@ -59,6 +59,10 @@
                   but return an empty string instead.
             type: bool
             default: false
+        extract:
+            description:
+                - Tell SOPS to extract a specific key from a json or yaml file.
```

Comment:
```suggestion
                - Tell SOPS to extract a specific key from a JSON or YAML file.
```
Also I would add some more paragraph(s) roughly explaining the syntax.